src/extras/webgl/TilingSpriteRenderer.js

```diff
@@ -130,17 +143,36 @@ export class TilingSpriteRenderer extends core.ObjectRenderer {
         shader.uniforms.uTransform = tempMat.toArray(true);
 
         const color = tempArray;
+        const alpha = ts.worldAlpha;
 
         core.utils.hex2rgb(ts.tint, color);
-        color[3] = ts.worldAlpha;
+        color[0] *= alpha;
+        color[1] *= alpha;
+        color[2] *= alpha;
+        color[3] = alpha;
```

Comment:
Well as far as I can tell this is the only place where we multiply in code and not in the shader, so I would say this breaks the current convention.